bedrock/firefox/templates/firefox/tiles.html

```diff
@@ -31,9 +31,6 @@
   {{ fxfamilynav('desktop', '', 'dark') }}
 {% endblock %}
 
-{% block tabzilla_css %}{% endblock %}
-{% block tabzilla_js %}{% endblock %}
-
```

Comment:
As noted above, as long as we change `dashboard-base.html` to `site_css`, let's change this page to `page_css`.